plugins/org.eclipse.fordiac.ide.model/src/org/eclipse/fordiac/ide/model/typelibrary/impl/SystemEntryImpl.java

```diff
@@ -80,4 +81,11 @@ protected AbstractTypeExporter getTypeExporter(final AutomationSystem type) {
 	public EClass getTypeEClass() {
 		return LibraryElementPackage.Literals.AUTOMATION_SYSTEM;
 	}
+
+	@Override
+	protected synchronized NotificationChain basicSetTypeEditable(final LibraryElement newTypeEditable,
+			final NotificationChain notifications) {
+		return super.basicSetType(newTypeEditable, notifications);
+	}
+
```

Comment:
I believe you also need to similarly override `basicGetTypeEditable()` for this to work correctly.

Reply:
ah yes you are right. It gets time that we get rid of the editable.